src/main/java/frc/robot/commands/ChargeCommand.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class ChargeCommand extends CommandBase {
+  ProfiledPIDController pidControlX;
+  double currentX;
+  double desiredX;
+  double deltaX;
+
+  double rollValue;
+  double initRollValue;
+
+  double ROLL_TOLERANCE = 15;
+  /** Creates a new ChargeCommand. */
+  public ChargeCommand( double deltaX) {
+    pidControlX = new ProfiledPIDController(1, 0.0, 0.2, 
+    new TrapezoidProfile.Constraints(1, 1));
```

Comment:
These will likely be tuned to be faster. We could start with the values from AlignToVisionTarget instead, since those are slighlty faster.